clang/lib/Sema/SemaExpr.cpp

```diff
@@ -4868,6 +4905,9 @@ ExprResult Sema::CreateUnaryExprOrTypeTraitExpr(TypeSourceInfo *TInfo,
 ExprResult
 Sema::CreateUnaryExprOrTypeTraitExpr(Expr *E, SourceLocation OpLoc,
                                      UnaryExprOrTypeTrait ExprKind) {
+  GetSymbolicName obj;
+  obj.VisitExpr(E);
+  StringRef LastString = obj.getLastString();
```

Comment:
you don't have to do this if you are not creating a `nameof` expr. I suggest you move this to the following `if(ExprKind == UETT_nameof)` block.